sda/cmd/download/swagger_v1.yml

```
@@ -0,0 +1,275 @@
+openapi: 3.0.3
+info:
+  title: SDA file download API
+  version: "1.0"
+paths:
+  /file:
+    get:
+      description: Returns the requested file to the user
+      parameters:
+        - in: header
+          description: "Public key used to re-encrypt the file (header) with. This should be supplied as the base64 encoding of the PEM (RFC 7468) encoding of the public key"
+          name: public_key
+          required: true
+          schema:
+            type: string
+        - in: header
+          description: "The JWT used to authenticate the user"
+          name: token
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: dataset
+          schema:
+            type: string
+          required: true
+          description: Stable ID of the dataset
+        - in: query
+          name: fileId
+          schema:
+            type: string
+          required: false
+          description: Stable ID of the file to download. It is an error to supply both filePath and fileId
+        - in: query
+          name: filePath
+          schema:
+            type: string
+          required: false
+          description: File path of the file to download. It is an error to supply both filePath and fileId
+        - in: query
+          name: end
+          description: 'End of requested data in the form of the 0-index of the first octet not included in the
+                                 response. That is, data delivered will have octet indices in the closed-open set
+                                 [start,end). Defaults to end of file if not set'
+          required: false
+          schema:
+            minimum: 0
+            type: integer
+        - in: query
+          name: start
+          required: false
+          schema:
+            default: 0
+            minimum: 0
+            type: integer
+      responses:
+        "200":
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          description: successful operation
+        "206":
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          description: successful partial file delivery
```

```suggestion
          description: Successful partial file delivery
```
Uppercasing the messages for 200 (here and below) for consistency.